Adapt therapeutics abstract for new format (#964)
greenelab/covid19-review: content/20.pharmaceuticals.md

### Abstract

> After emerging in China in late 2019, the novel _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021, continues to significantly impact most countries.

Reviewer: ```suggestion
After emerging in China in late 2019, the novel _severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021, continues to significantly impact most countries.
```

> Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.

Reviewer: ```suggestion
Only a small number of coronaviruses are known to infect humans, and only two are associated with the severe outcomes associated with SARS-CoV-2: _severe acute respiratory syndrome coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome coronavirus_, which emerged in 2012.
```

Reviewer: You can correct me if I'm wrong but no need for the related part I think

Author: I'm sorry I'm always shooting you down on this but I think this is how they want it, based on their Excel sheet of virus names! I'm so sorry that I never incorporate these changes, we'll see if my interpretation of the style guide is correct 😂

Reviewer: Haha this is funny! No problem, you know I will totally forget and say it again haha

Both of these previous epidemics were controlled fairly rapidly through public health measures.

> However, previous insights into the immune response to coronaviruses gained during the outbreaks of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) have proved beneficial to identifying approaches to the treatment and prophylaxis of novel coronavirus disease 2019 (COVID-19).

Reviewer: ```suggestion
However, previous insights into the immune response to coronaviruses gained during the outbreaks of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) have proved beneficial for identifying approaches to treat and prevent novel coronavirus disease 2019 (COVID-19).
```
